### PR TITLE
stepper: add the comment to link the specification

### PR DESCRIPTION
**Pull request by FYL2003**
The change is uploaded.

### CODE REVIEW

File: src/stepper/stepper.ts [hunk 1]
```diff
@@ -3361,6 +3369,10 @@ export function getEvaluationSteps(
       [],
       'Start of evaluation'
     ]
+    /**
+     * push the content into frontend for showing contents in the
```
Comment (martin-henz) [inline src/stepper/stepper.ts:3373]: This sentence is not complete: "in the" ...

File: src/stepper/stepper.ts [hunk 2]
```diff
@@ -3361,6 +3369,10 @@ export function getEvaluationSteps(
...
+     * this comment may not be correct, please correct it if you find any mismatch in description
+     */
```
Comment (martin-henz) [inline src/stepper/stepper.ts:3375]: This comment "this comment may not be correct" is quite strange. Are you referring to the one above: "push the content..."?

File: src/stepper/stepper.ts [hunk 3]
```diff
@@ -3405,6 +3422,10 @@ export function getEvaluationSteps(
         ])
       }
     }
+    /**
+     * Deprecated check for empty program
+     * It can be deleted or kept as a reference for legacy code.
+     */
```
Comment (martin-henz) [inline src/stepper/stepper.ts:3428]: If the check is deprecated, please delete it.

**Review by martin-henz — CHANGES_REQUESTED** [review]

**Review by RichDom2185 — COMMENTED** [review]
Just a minor nit: the comments should be either

```js
// line 1
// line 2
```

or `/* ... */` multi-line comments, not `/** .... */` documentation comments.

**Review by martin-henz — APPROVED** [review]

**Pull request merged**